kvbc/include/kvbc_adapter/replica_adapter.hpp

```diff
@@ -223,6 +223,9 @@ class ReplicaBlockchain : public IBlocksDeleter,
   const concord::kvbc::categorization::KeyValueBlockchain *const getReadOnlyCategorizedBlockchain() {
     return kvbc_.get();
   }
+  std::map<std::string, std::vector<std::string>> getBlockStaleKeys(BlockId block_id) const {
+    return kvbc_ == nullptr ? v4_kvbc_->getBlockStaleKeys(block_id) : kvbc_->getBlockStaleKeys(block_id);
```

Comment:
This check can be avoided at the adapter layer if we use an interface. In the long run interface will help when there will be many implementations.

Reply:
you're right, but this functionality is so insignificant, I don't think it worth the change now